simulator/index.md

```diff
@@ -77,35 +69,6 @@ Within the Webots IDE, there are a few different panels:
 - [Camera Controls](https://www.cyberbotics.com/doc/guide/the-3d-window#navigation-in-the-scene)
 - [Graphics settings](https://www.cyberbotics.com/doc/guide/preferences#opengl) (Useful if Webots is running slowly)
 
-## Robot
```

Comment:
Thought: It'd be nice to keep this section, or as much of it as possible. Probably just linking out to the programming page for the sensor reference.

Reply:
It makes more sense to talk about the robot in the context of its attached sensors.